packages/chart/src/Pie.ts

```diff
@@ -52,10 +54,22 @@ export class Pie extends SVGWidget {
     }
 
     calcOuterRadius() {
-        const maxTextWidth = this.textSize(this.data().map(d => d[0]), "Verdana", 12).width;
+        const maxTextWidth = this.textSize(this.data().map(d => this.getLabelText({ data: d })), "Verdana", 12).width;
         return Math.min(this._size.width - maxTextWidth * 2 - 20, this._size.height - 12 * 3) / 2 - 2;
     }
-
+    calcTotalValue(): number {
+        return this.d3Pie(this.data()).reduce((acc, d) => {
+            return acc + d.data[1];
+        }, 0);
```

Comment:
I don't think there is any need to run the data through a d3Pie to simply get a total?

Reply:
@GordonSmith this has been resolved